#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_no-connect-test.patch by Stefan Lippers-Hollmann <s.l-h@gmx.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: If localhost isn't already in known_hosts, the connect test will fail.

@DPATCH@
diff -Nrup --exclude=debian freenx-0.4.5.orig/nxsetup freenx-0.4.5/nxsetup
--- freenx-0.4.5.orig/nxsetup	2005-09-03 20:24:01.000000000 +0200
+++ freenx-0.4.5/nxsetup	2005-09-04 01:21:34.911286750 +0200
@@ -431,7 +431,7 @@ then
 	
 	install_nx
 
-	test_nx
+	[ "$AUTOMATIC" = "no" ] && test_nx
 	
 	echo "Ok, nxserver is ready."
 	echo 
